readme.md

```diff
@@ -535,6 +535,7 @@
 - [Web Design](https://github.com/nicolesaidy/awesome-web-design) - For digital designers.
 - [JMeter](https://github.com/aliesbelik/awesome-jmeter) - Load testing and performance measurement tool.
 - [Creative Coding](https://github.com/terkelg/awesome-creative-coding) - Programming something expressive instead of something functional.
+- [Livecoding](https://github.com/lvm/awesome-livecoding) - All things livecoding.
```

Comment:
> The entry in the Awesome list should:
Include a short description about the project/theme of the list. **It should not describe the list itself.**

:arrow_up: